static/app/components/events/autofix/autofixRootCause.tsx

```diff
@@ -565,3 +607,41 @@ const RootCauseOptionsRow = styled('div')`
   display: flex;
   flex-direction: row;
 `;
+
+const fadeIn = keyframes`
+  from { opacity: 0; }
+  to { opacity: 1; }
+`;
+
+const StyledIconWrapper = styled('span')`
+  color: inherit;
+  line-height: 0;
+`;
+
+const LinkStyles = css`
+  align-items: center;
+  gap: ${space(0.75)};
+`;
+
+const OpenInLink = styled(ExternalLink)`
+  ${LinkStyles}
+  color: ${p => p.theme.subText};
+  animation: ${fadeIn} 0.2s ease-in-out forwards;
```

Comment:
Do you need this to fade in?

Reply:
I just did that to match the style used in `stacktraceLink.tsx` for consistency.